perfdash/perfdash.go

```diff
@@ -107,7 +113,7 @@ func parseTestOutput(scanner *bufio.Scanner, buildNumber int, resources sets.Str
 			if strings.Contains(line, "INFO") || strings.Contains(line, "STEP") || strings.Contains(line, "Failure") || strings.Contains(line, "[AfterEach]") {
 				obj := LatencyData{}
 				if err := json.Unmarshal(buff.Bytes(), &obj); err != nil {
-					fmt.Printf("error parsing JSON in build %d: %v %s\n", buildNumber, err, buff.String())
+					fmt.Errorf("error parsing JSON in build %d: %v %s\n", buildNumber, err, buff.String())
```

Comment:
fmt.Errorf() doesn't log anything (it just creates an error, which you are dropping here) - so this change is incorrect